plugins/stack-overflow-backend/src/search/StackOverflowQuestionsCollatorFactory.ts

```diff
@@ -81,7 +81,11 @@ export class StackOverflowQuestionsCollatorFactory
     this.apiAccessToken = options.apiAccessToken;
     this.teamName = options.teamName;
     this.maxPage = options.maxPage;
-    this.requestParams = options.requestParams;
+    this.requestParams = options.requestParams ?? {
+      tagged: ['backstage'],
+      site: 'stackoverflow',
+      pagesize: 100,
+    };
```

Comment:
Open for suggestions on a different default value 🙂

Reply:
This is proably best answered by @emmaindal .

But it raises the question of, do people now need to know to always put in a few specific things in this config field every time? What happens if you leave out `site` for example?

Should there be some form of merging happening? Should this be

```ts
this.requestParams = {
      tagged: ['backstage'],
      site: 'stackoverflow',
      pagesize: 100,
      ...(options.requestParams ?? {})
    };
```

or something like that?

Reply:
Update to use the same default params passed in the official Stack Overflow documentation 🙌🏻 : https://github.com/backstage/backstage/pull/20456/commits/620ccbc728a1447c295ff270399cdc726dee181a